_posts/2017-07-14-Rust-Roadmap-Update.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: post
+title: "Rust Roadmap Update"
+author: Nicholas Matsakis
+---
+
+In January of this year, we adopted the [2017 Rust Roadmap][rr], which
+laid out our plans for 2017. Now that 2017 is roughly half over, we
+thought we would check in on our progress in each of our roadmap areas
+thus far.  In addition to talking about what's been going on, we'll
+also include some notes on how you can get involved.
+
+[rr]: https://github.com/rust-lang/rfcs/blob/master/text/1774-roadmap-2017.md
+
+### Tracking progress on the roadmap
+
+First, a meta note. If you'd like to follow along with the progress on
+a particular roadmap initiative, or to find out more about how you can
+get involved, the best place to go is the
+[list of issues on the rust-roadmap repo](https://github.com/rust-lang/rust-roadmap/issues/).
+There you will find an issue dedicated to each of the major
+initiatives that we are pushing on. These issues contain links to
+ongoing work. You'll find a number of links to issues like this in the
+descriptions that follow.
+
+### Rust should have a lower learning curve
+
+The most direct way to make Rust easier to learn is to improve the way
+that we teach it. To that end, we've been hard at work on a brand new
+edition of the "Rust" book ([roadmap issue][rr7]), and we now have a
+[complete draft available online][book]. This new edition puts
+ownership front and center and it also has expanded coverage of a
+number of other areas in Rust, such as error handling, testing,
+matching, modules, and more. Even better, you can
+[pre-order a printed version][preorder] through No Starch Press. If
+you'd like to help out, there is still
+[lots of editing work to be done!](https://github.com/rust-lang/book/projects/1)
+
+[rr7]: https://github.com/rust-lang/rust-roadmap/issues/7
+[preorder]: https://www.nostarch.com/rust
+[book]: https://doc.rust-lang.org/book/
+
+We've also been working on a number of language changes aimed at
+improving [language ergonomics][ergo]. These range from long-standing
+proposals, like [non-lexical lifetimes][rr16] or [`impl Trait`], to
+newer ideas, like the recently approved RFCs on [trait aliases][] and
+[match ergonomics][]. On the [roadmap issue][rr17], you will find a
+large list of initiatives, organized by the part of the language that
+they target (e.g., ownwership/borrowing, the trait system, etc). We
```

Comment:
"ownwership" is "own worship"? Rust Evangelism Strike Force are actually solipsists?!!